core/primitives/src/version.rs

```diff
@@ -164,6 +164,29 @@ pub const PROTOCOL_VERSION: ProtocolVersion = MAIN_NET_PROTOCOL_VERSION;
 #[cfg(feature = "nightly_protocol")]
 pub const PROTOCOL_VERSION: ProtocolVersion = 126;
 
+#[allow(dead_code)]
+const MAIN_NET_PROTOCOL_UPGRADE_VOTING_START: Option<ProtocolUpgradeVotingSchedule> =
+    Some(ProtocolUpgradeVotingSchedule("2022-02-17 12:44:00"));
```

Comment:
what is this hardcoded number?

Reply:
Arbitrary point in time used for testing. Removed.